source/init/sentry.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import {SENTRY_DSN} from './constants'
+import {Sentry, SentryLog} from 'react-native-sentry'
+import {IS_PRODUCTION} from '@frogpond/constants'
+
+export function bootSentry() {
+	let config = Sentry.config(SENTRY_DSN, {
+		logLevel: SentryLog.Debug,
+		handlePromiseRejection: true,
+	})
+
+	if (!IS_PRODUCTION) config.install()
```

Comment:
Make this if multi-line please

Reply:
Prettier did this.

Reply:
Put braces around it and prettier will leave it multi-line